gusto/equations.py

```diff
@@ -484,36 +485,37 @@ def generate_tracer_transport_terms(self, active_tracers):
         return adv_form
 
 
-class ForcedAdvectionEquation(PrognosticEquationSet):
+class CoupledTransportEquation(PrognosticEquationSet):
     u"""
-    Discretises the advection equation with a source/sink term,               \n
+    Discretises a transport equation with a source/sink term,               \n
     ∂q/∂t + (u.∇)q = F,
-    which can also be augmented with active tracers.
+    which uses multiple active tracers.
+    In this class, we have multiple tracers or species that are
+    interacting, so q and F are vectors.
+
     """
-    def __init__(self, domain, function_space, field_name, Vu=None,
-                 active_tracers=None, **kwargs):
+    def __init__(self, domain, active_tracers, Vu=None):
         """
         Args:
             domain (:class:`Domain`): the model's domain object, containing the
                 mesh and the compatible function spaces.
-            function_space (:class:`FunctionSpace`): the function space that the
-                equation's prognostic is defined on.
-            field_name (str): name of the prognostic field.
+            active_tracers (list): a list of `ActiveTracer` objects
+                that encode the metadata for any active tracers to be included
+                in the equations. This is required for using this class; if there
+                is only a field to be advected, use ForcedAvectionEquation()
```

Comment:
So I think we're now wondering about not having a `ForcedAdvectionEquation`, so this could say user "AdvectionEquation" or "TransportEquation" instead